app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
```

Comment:
I don't think we should deliver a proguarded version of the library. It personal drives me insane if I can't `Ctrl + B` a method and get the uncompiled/unobfuscated raw source code.

We have `minifyEnabled false` in FirebaseUI with a `consumerProguardFile` so the dev can decide to obfuscate their app if they want, but still develop it with the library source code.